### PR TITLE
Add a test

### PR DESCRIPTION
**Pull request by matthew-andrews**

### CODE REVIEW

File: test/main.test.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+		};
+
+		main.__set__('dynamos', {
+			master: { table: 'urlmgmtapi_master', instance: mockInstance },
+			slave: { table: 'urlmgmtapi_slave', instance: mockInstance }
+		});
+	});
+
+	it('should #get /fastft', () => {
+		return main.get('www.ft.com/fastft')
```
Comment (wheresrhys) [inline test/main.test.js]:
```
it('should #get /fastft', () => main.get('www.ft.com/fastft')
```

Comment (matthew-andrews) [inline test/main.test.js]: so hiipster

Comment (matthew-andrews) [inline test/main.test.js]: hmm i think that makes it less readable

Comment (wheresrhys) [inline test/main.test.js]: I misunderstood the original request. I can't decide which I prefer

**Pull request merged**